modules/administration/pages/tshoot-sync.adoc

```diff
@@ -45,6 +45,17 @@ RepoMDError: Cannot access repository. Maybe repository GPG keys are not importe
 To resolve the problem, you need to import the GPG key to {productname}.
 For more on importing GPG keys, see xref:administration:repo-metadata.adoc[].
 
+
+GPG Key Removal::
```

Comment:
I would consider here a different title to avoid messing with removing GPG keys in any other context.

I would suggest something like:

```suggestion
GPG Key Removal from "spacewalk-repo-sync"::
```